### ISSUE TITLE
New UI truncates account names

### ISSUE THREAD

**Issue by ncs-alexhoward** [WTFender/aws-sso-extender#64]
### Extension Version

1.6.0

### Description

The new UI truncates the account name so it can be a bit tricky when there are a number of accounts that start with the same initial string.

### Browsers

Firefox

### OS

Mac

**Comment by WTFender** [comment]
Agreed; need to handle dynamic width a bit better & will get it fixed up shortly.

I'm considering the ability to rename (label) accounts/apps too.

**Comment by ncs-alexhoward** [comment]
Labeling would be great. Thank you!

**Comment by WTFender** [comment]
Latest version should have better sizing and spacing. Let me know if it's still a problem.

**Comment by ncs-alexhoward** [comment]
This issue seems to have resurfaced with the latest release (1.8.11).
